aries-site/src/examples/templates/inline-notifications/PageBannerExample.js

```diff
@@ -43,7 +43,7 @@ export const PageBannerExample = () => {
             // changes the route and sets the active tab.
             actions={[{ label: 'View details', onClick: () => setActive(2) }]}
           />
-          <Tabs activeIndex={active}>
+          <Tabs justify="start" activeIndex={active}>
```

Comment:
@taysea @britt6612 
I'm wondering for HPE examples if there is any preference for `alignControls="start"` vs `justify="start"` or can they be used interchangeably?

Reply:
- [DS documentation](https://design-system.hpe.design/components/tabs?q=tab#alignment) suggests using `justify="start"`

- [Grommet documentation](https://v2.grommet.io/tabs#alignControls) suggests:
  - `justify` -> To align the tabs along the main axis.
  - `alignControls` -> To align the tab controls along the vertical axis.